### DISCUSSION TITLE
Draw line using Haversine formula

### DISCUSSION THREAD

**Discussion by ibragimof** [Leaflet/Leaflet#7807] [Q&A] [answered]
Leaflet in its method `distanceTo()` [uses](https://leafletjs.com/reference.html#latlng-distanceto) "[Haversine formula](https://en.wikipedia.org/wiki/Haversine_formula)"  to get actual distance on a earth planet's sphere. In cases where we'll put two markers in large distance from each other, it's very important take into account the sphericity of the earth.

Despite this, Leaflet's method `L.polyline` [draws ](https://leafletjs.com/reference.html#polyline-l-polyline) all its path through a direct connection of two markers.

How we can get some bent appearing on drawing lines (due to the curvature of the earth)?

**Comment by Falke-Design** [comment, marked as answer]
Please look into the available [plugins](https://leafletjs.com/plugins.html#geolocation). I know there is a plugin but I don't know the name anymore

**Reply by IvanSanchez** [reply]
That'd be https://github.com/henrythasler/Leaflet.Geodesic

**Comment by johnd0e** [comment]
Another plugin: https://github.com/IITC-CE/Leaflet.Geodesic
